docs/eth/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: What is ETH?
+meta:
+  - name: description
+    content: The basic information you need to understand ETH.
+  - property: og:title
+    content: What is ETH?
+  - property: og:description
+    content: The basic information you need to understand ETH.
+lang: en-US
+---
+
+# What is ETH?
+
+<div class="featured">
+
+  **ETH is the native currency of Ethereum** This page gives you the basic information you need to get started using ETH.
+
+</div>
+
+
+## What is ETH, and how do I get it?
+
+Ether, or ETH, is Ethereum's native currency. It is "digital money" that can be sent over the internet instantly and cheaply, and also be used in many Ethereum-based applications like the ones above.
```

Comment:
`[Ethereum-based applications](/dapps)`

Also, we should cut `like the ones above`.